src/paperless/settings.py

```diff
@@ -460,6 +460,13 @@ def default_threads_per_worker(task_workers):
 
 CONSUMER_SUBDIRS_AS_TAGS = __get_boolean("PAPERLESS_CONSUMER_SUBDIRS_AS_TAGS")
 
+CONSUMER_ENABLE_BARCODES = __get_boolean(
+    "PAPERLESS_CONSUMER_ENABLE_BARCODES",
+    default="YES",
```

Comment:
Since this is new functionality and I'm not sure how much use it will get, I'd prefer to see this default to No/False.  We wouldn't want to start splitting if the user doesn't want us to.

Reply:
Yea I debated asking for this too. If the feature works properly it shouldn't disturb documents without barcodes—right? But the other issue is if it slows down consumption even when no barcode is found. I assume it would, in which case I agree default should be `No`.

Reply:
I assumed it would not be on by default when I created the discussion point.  I like where this is going, and am hoping to start testing soon.

Reply:
I set the default to no, for now. We can still change it later, if people adopt to it. It shouldn't delay anything for people without `PATCHT` barcodes, though.

Reply:
Well, you rasterize all pages looking for those barcodes. That will slow down processing even if you find nothing.